src/ci/bin/testCentaurAws.sh

```diff
@@ -18,29 +18,39 @@ export AWS_CONFIG_FILE="${CROMWELL_BUILD_RESOURCES_DIRECTORY}"/aws_config
 # "space" and "scatter" tests are disabled because hey intermittently fail on AWS
 # https://broadworkbench.atlassian.net/browse/BA-6152
 # NOTE! For some reason, exclusions must all be lower case even if the test name is a mixture of upper and lower
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# !!!!!!!!!! https://broadworkbench.atlassian.net/browse/BT-57 !!!!!!!!!!
+# !!!!!!!!!! DISABLING ALL TESTS BUT HELLO WORLD (Travis) AND  !!!!!!!!!!
+# !!!!!!!!!!   MUTECT2 (Jenkins) UNTIL AWS CAN AUTHENTICATE    !!!!!!!!!!
+# !!!!!!!!!!                WITH DOCKER REPOS                  !!!!!!!!!!
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 cromwell::build::run_centaur \
-    -p 100 \
-    -e smartseq2singlesample \
-    -e arrays \
-    -e haplotypecaller \
-    -e jointdiscovery \
-    -e mutect2 \
-    -e singlesample \
-    -e singlesample_production \
-    -e cnv_somatic_pair \
-    -e cnv_somatic_panel \
-    -e localdockertest \
-    -e non_root_default_user \
-    -e non_root_specified_user \
-    -e abort.scheduled_abort \
-    -e relative_output_paths \
-    -e relative_output_paths_colliding \
-    -e standard_output_paths_colliding_prevented \
-    -e space \
-    -e scatter \
-    -e long_cmd \
-    -e runtwiceexpectingcallcaching \
-    -e cwl_cache_within_workflow \
-    -e cachewithinwf
+     -i hello \
+     -i mutect2.aws
+
+#    -p 100 \
+#    -e smartseq2singlesample \
+#    -e arrays \
+#    -e haplotypecaller \
+#    -e jointdiscovery \
+#    -e mutect2 \
+#    -e singlesample \
+#    -e singlesample_production \
+#    -e cnv_somatic_pair \
+#    -e cnv_somatic_panel \
+#    -e localdockertest \
+#    -e non_root_default_user \
+#    -e non_root_specified_user \
+#    -e abort.scheduled_abort \
+#    -e relative_output_paths \
+#    -e relative_output_paths_colliding \
+#    -e standard_output_paths_colliding_prevented \
+#    -e space \
+#    -e scatter \
+#    -e long_cmd \
+#    -e runtwiceexpectingcallcaching \
+#    -e cwl_cache_within_workflow \
+#    -e cachewithinwf
```

Comment:
ToL: I'm not sure we need this commented out code if we have git histories.

Reply:
FWIW, since we know we're going to be reviving this later, I much prefer having it commented out rather than having to fish though git history for it.

Reply:
Also, as a self-aware pack rat, I appreciate you for raising the question!